2 - RTB/Documentazione esterna/Analisi dei Requisiti/Analisi dei Requisiti.typ

```diff
@@ -1131,7 +1131,13 @@ Dove:
     
     [FM-12], [Obbligatorio], [Il prodotto deve essere ad accesso pubblico, ovvero senza login], [Capitolato],
 
-    [FM-13], [Obbligatorio], [Il prodotto deve prevedere una sola tipologia di utente], [Capitolato]
+    [FM-13], [Obbligatorio], [Il prodotto deve prevedere una sola tipologia di utente], [Capitolato],
+
+    [FM-14], [Obbligatorio], [Al termine della sessione di lavoro il prodotto non deve salvare la configurazione dell'ambiente creata dall'utente], [Capitolato],
+    [FM-14.1], [Obbligatorio], [Al termine della sessione di lavoro il prodotto non deve salvare le modifiche effettuate], [Capitolato],
+    [FM-14.1.1], [Obbligatorio], [Al termine della sessione di lavoro il prodotto non deve fornire il salvataggio delle modifiche effettuate agli scaffali], [Capitolato],
```

Comment:
FM-14.1.x mi sembrano un po' ripetitivi. Se invece fossero:

```
Il prodotto non persiste in locale (cookie, `localStorage`) le modifiche fatte all'ambiente

Il prodotto non persiste sul database le modifiche fatte all'ambiente

Il prodotto si deve avviare allo stato iniziale ogni volta che viene ricaricata la pagina

Il prodotto non deve fornire alcuna opzione per il salvataggio dei dati
```

come li vedi?

Reply:
Non so eh

Reply:
anche secondo me sono meglio le opzioni di Giovanni. Ma a questo punto modificherei la gerarchia dei requisiti, per esempio: "FM-14 Il prodotto si deve avviare allo stato iniziale ogni volta che viene ricaricata la pagina", "FM-14.1 Il prodotto non persiste in locale (cookie, `localStorage`) le modifiche fatte all'ambiente", "FM-14.2 Il prodotto non persiste sul database le modifiche fatte all'ambiente", "FM-14.3 Il prodotto non deve fornire alcuna opzione per il salvataggio dei dati"